util/generate-ami-list.py

```diff
@@ -37,29 +37,49 @@
         ("ubuntu1804", "ubuntu-1804"),
     ]
 )
+ARCHITECTURES_TO_MAPPING_NAME = {
+    "x86_64": "AWSRegionOS2AMIx86",
+    "arm64": "AWSRegionOS2AMIarm64",
+}
+
+
+def get_initialized_mappings_dicts():
```

Comment:
my understanding is that we were moving away from that, especially for standalone scripts.

Reply:
I think we should still following the style guide here: https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles, and use `_` prefix for weak internal use indication in files that might be imported by others.

For this file I am ok with keeping as is because we are unlikely to use functions here elsewhere